Sources/Client/Views/Settings/VideoSettingsView.swift

```diff
@@ -7,13 +7,16 @@ struct VideoSettingsView: View {
   
   @State var renderDistance: Float = 0
   @State var fov: Float = 0
+  @State var upscaleFactor: Float = 0
```

Comment:
Shouldn't 1 be the default scaling factor?

Reply:
I went with 0 because 1 would indicate to me that we are upscaling texture from native to native size. Default upscaling  set to 0 indicates that there is no upscaling performed at all.

Reply:
If it's indicating none, then you should make it an optional, because a scaling factor of 0 sounds to me like it's going to scale to 0 width and 0 height (multiply by zero). I think 1 makes sense though anyway because even if upscaling isn't enabled we're still 'upscaling' from native size to native size when rendering the screen, that's just a no-op so no scaling actually happens.

Reply:
Exactly. With upscaling factor equal 0 you are scaling width and height by 0 = you don't have any upscaling. Also, with step 2 of UI slider, having 0-2-4 factors makes more sense than 1-2-4. I'll remove it for now, it's unused anyways.